Makefile

```
@@ -148,51 +146,22 @@ test: core
 
 ########################## MAKE/INSTALL RULES ##########################
 #
-all: dependencies core
+all: core
 
 core:
 	@command -v $(firstword $(MSBUILD)) >/dev/null || (echo "OpenRA requires the '$(MSBUILD)' tool provided by Mono >= 5.4."; exit 1)
 ifeq ($(WIN32), $(filter $(WIN32),true yes y on 1))
-	@$(MSBUILD) -t:build -p:Configuration="Release-x86"
+	@$(MSBUILD) -t:build -p:Configuration="Release-x86" -p:TargetPlatform=$(TARGETPLATFORM) 
 else
-	@$(MSBUILD) -t:build -p:Configuration=Release
+	@$(MSBUILD) -t:build -p:Configuration=Release  -p:TargetPlatform=$(TARGETPLATFORM)
 endif
 
 clean:
 	@ $(MSBUILD) -t:clean
 	@-$(RM_F) *.config
 	@-$(RM_F) *.exe *.dll *.dylib ./OpenRA*/*.dll *.pdb mods/**/*.dll mods/**/*.pdb *.resources
 	@-$(RM_RF) ./*/bin ./*/obj
```

Changing the order here at least fixes `make clean` on my end.

```suggestion
	@-$(RM_F) *.config
	@-$(RM_F) *.exe *.dll *.dylib ./OpenRA*/*.dll *.pdb mods/**/*.dll mods/**/*.pdb *.resources
	@-$(RM_RF) ./*/bin ./*/obj
	@ $(MSBUILD) -t:clean
```